core/src/main/scala/io/finch/request/package.scala

```diff
@@ -162,6 +180,8 @@ package object request {
       def apply[Req](req: Req)(implicit ev: Req => HttpRequest) =
         try number.toFuture
         catch { case _: IllegalArgumentException => ValidationFailed(param, rule).toFutureException }
+
+      override def toString: String = s"param: $param"
```

Comment:
I was thinking about using `ValidationFailed` here. I was hoping that we can keep the feature of representing the error as pair: `item` (previously param) and `rule`. I found this feature pretty useful, since we're able to pattern-match it send the `item` information wrapped in JSON, so clients may easily parse and visualize it. Do you think it's possible to continue sending `ValidationFailed` from new `ValidationRule` with both `item` and `rule`. The frontend teams will be happy to keep this functionality.